front/lib/api/files/upload.ts

```diff
@@ -347,6 +347,12 @@ const processingPerContentType: ProcessingPerContentType = {
     avatar: notSupportedError,
     tool_output: notSupportedError,
   },
+  "dust-application/slack": {
+    conversation: storeRawText,
+    folder: storeRawText,
```

Comment:
nit: I would say "folder" not supported too.

Reply:
fair